content/projects/tdd/password-checker/part1/_index.md

```diff
@@ -149,4 +149,13 @@ Please don't re-implement the same check in two different places. Good code is D
 
 ## Java Resources
 
-- https://howtodoinjava.com/junit5/expected-exception-example/
\ No newline at end of file
+- https://howtodoinjava.com/junit5/expected-exception-example/
+
+## Instructions for reviewers
+
+- **Error messages** - Make sure the passwordIsValid function throws the relevant error messages and should match the 7 conditions above. Use a seperate file for your error messages.
+
+- **Count conditions and what to return** - For passwordStrength function, if condition 1 and 2 is not met and all the other conditions are met then the function should return Invalid. e.g. if I pass in `P@ssw 12` then the function should return Invalid cause condition 2 is not met `password should be longer than 8 characters`.
+
+- **Testing** - Make sure that the spec file contains multiple test cases that are not short and checks if every condition is working correctly for both functions and the relevant error messages are thrown. The tests should prove that the functions are working correctly and that the code does not break. e.g.
+`P$$12 A` or `User1@` should fail for both functions, for PasswordIsValid the relevant error message should be thrown, and for passwordStrength `Invalid` should be returned.
```

Comment:
This also seems to be paraphrasing the question. We already said that "Invalid" should be returned, didn't we? 

The student must always follow the instructions. Repeating the instructions is not DRY.

Then when it comes to talking about unit testiong it might actually make sense to put that stuff at the top of the file. Something like:

"Please note that this is a uniit testing project. Make sure you demonstrate solid understanding of unit testing. "